.gitignore

```diff
@@ -83,3 +83,6 @@ yalc.lock
 ## Documentation Report
 scripts/docs-report.md
 
+# Temporary data collected by Million Lint
+**/.million/store.json
+dev/test-studio/.react-compiler-bailout-report.json
```

Comment:
Committing this to git would cause all sorts of annoying merge conflicts on PRs that do perf work, that for example makes components no longer bail out. Or when `eslint-plugin-react-compiler` and `babel-plugin-react-compiler` itself changes what cases it bails and supports as we bump deps.